tests/utils.go

```diff
@@ -2071,21 +2066,17 @@ func NewRandomVirtualMachineInstanceWithBlockDisk(imageUrl, namespace string, ac
 	return NewRandomVirtualMachineInstanceWithDisk(imageUrl, namespace, sc, accessMode, k8sv1.PersistentVolumeBlock)
 }
 
-func NewRandomDataVolumeWithRegistryImportInStorageClass(imageUrl, namespace, storageClass string, accessMode k8sv1.PersistentVolumeAccessMode, volumeMode k8sv1.PersistentVolumeMode) *cdiv1.DataVolume {
-	name := TestDataVolumeName + rand.String(12)
-	quantity, err := resource.ParseQuantity("1Gi")
+func newDataVolumeWithManyOptions(namespace, storageClass string, size string, accessMode k8sv1.PersistentVolumeAccessMode, volumeMode k8sv1.PersistentVolumeMode, dataVolumeSource cdiv1.DataVolumeSource) *cdiv1.DataVolume {
```

Comment:
does it really need the "ManyOptions" suffix?

Reply:
naming things is hard. Do you suggest a simple `newDataVolume` ?

Reply:
yes, I think that would be enough (unless we need to differentiate from another newDataVolume function)

Reply:
We have other `newDataVolume` entities elsewhere, but not in the `tests` package.